.github/workflows/ci-tests.yml

```diff
@@ -111,7 +111,7 @@ jobs:
       - name: "nox cache"
         uses: ./.github/workflows/composite/nox-cache
         with:
-          cache_build: 0
+          cache_build: 1
```

Comment:
Thanks to confusion in `noxfile.py`, our `3.10` tests have actually been testing with `3.8` until now! So I needed to get a new Nox environment that was actually `3.10`.